src/test/java/org/wiremock/RandomHelperTest.java

```diff
@@ -0,0 +1,47 @@
+package org.wiremock;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.any;
+import static org.hamcrest.Matchers.is;
+
+import com.github.jknack.handlebars.Helper;
+import com.github.jknack.handlebars.Options;
+import java.io.IOException;
+import java.util.ArrayList;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class RandomHelperTest {
+  RandomHelper helper;
+  protected static final String FAIL_GRACEFULLY_MSG =
+      "Handlebars helper should fail gracefully and show the issue directly in the response.";
+
+  @BeforeEach
+  public void init() {
+    helper = new RandomHelper();
+  }
+
+  @Test
+  public void rendersAMeaningfulErrorWhenExpressionIsInvalid() {
+    try {
+      assertThat(
+          (String) renderHelperValue(helper, "something really random"),
+          is("[ERROR: Unable to evaluate the expression something really random]"));
+    } catch (final IOException e) {
+      Assertions.fail(FAIL_GRACEFULLY_MSG);
+    }
+  }
+
+  @Test
+  public void returnsRandomValue() throws Exception {
+    assertThat(renderHelperValue(helper, "Name.firstName"), is(any(String.class)));
```

Comment:
I'm wondering if there's a way we can show that we're doing the lookup right and getting something like what we asked for.

It might be possible to pin the random seed used by `Faker` and therefore get a predictable result each time that you could assert exactly.

Reply:
Added the seeded version of this test.